package.json

```diff
@@ -38,33 +38,35 @@
     "@aws-sdk/client-ssm": "^3.145.0",
     "@buildwithsygma/sygma-contracts": "2.3.0",
     "@celo-tools/celo-ethers-wrapper": "^0.0.5",
+    "@chainsafe/chainbridge-contracts": "3.1.0",
+    "@ethersproject/bignumber": "^5.7.0",
     "@ethersproject/providers": "^5.7.2",
-    "@fastify/compress": "6.2.1",
-    "@fastify/cors": "^8.2.1",
+    "@openzeppelin/contracts": "^4.8.3",
     "@polkadot/api": "^10.6.1",
     "@prisma/client": "^4.1.0",
+    "@types/mongodb": "^4.0.7",
     "cors": "^2.8.5",
     "docker-secret": "^1.2.4",
     "dotenv": "^10.0.0",
     "dotenv-flow": "^3.2.0",
-    "ethers": "^5.4.6",
-    "fastify": "4.17.0",
-    "fastify-healthcheck": "^4.4.0",
+    "ethers": "^6.3.0",
```

Comment:
@MakMuftic @mpetrun5 should we use v5.x for ethers? If we do, this will require to change the connection on the indexer. If we mantain this, we would need to change connection logic in the seed

Reply:
Update: I have adapted the other code to use v6. It is not worth the effort to revert considering the amount of code on the indexer that depends of the new API from v6